### DISCUSSION TITLE
How to add back the notification button

### DISCUSSION THREAD

**Discussion by Turhvjbufv** [anddea/revanced-patches#428] [Q&A] [answered]
![Screenshot_20240506-121030_ReVanced Extended_1_1.png](https://github.com/anddea/revanced-patches/assets/91851160/09a8a37b-8fe5-4aec-969a-faa377fe1238)

Can't seem to find it in settings (and it's not down there in the navigation bar)

**Comment by anddea** [comment, marked as answer]
Extended > General > Hide buttons in toolbar.
